stack.yaml

```diff
@@ -65,5 +65,5 @@ packages:
 # Allow a newer minor version of GHC than the snapshot specifies
 # compiler-check: newer-minor
 nix:
-  shell-file: scripts/nix/stack-shell.nix
+  shell-file: release.nix
```

Comment:
The convention from `cardano-chain` is to use a `scripts` folder and from `cardano-sl` is to use a `pkgs` folder.
I agree that there is no need to pollute the root directory so we can leave it there.

Reply:
Agree, lets minimize root folder nix scripts. We should be able to get it down to release.nix and shell.nix.